### ISSUE TITLE
Unhandled Promise rejection: No provider for ApplicationRef

### ISSUE THREAD

**Issue by jiker-burce** [MitocGroup/deep-microservices-todomvc#144]
OS sytem: Mac OS 10.12.06
node 6.10.3
Alter running the command: deepify server .
there is no error in the shell, the following errors appear in the chrome console.
but I saw this “Running "server after init hook" in deep-root-angular”， and I did it, but i just got the result like that:
SHELL:
```
➜  deep-root-angular git:(master) ✗ server after init hook
zsh: command not found: server
```
Chrome console:
```
zone.js:522 Unhandled Promise rejection: No provider for ApplicationRef! ; Zone: <root> ; Task: Promise.then ; Value: Error Error: No provider for ApplicationRef!
    at Error.ZoneAwareError (http://localhost:8000/js/polyfills.js:5190:33)
    at ZoneAwareError (http://localhost:8000/js/polyfills.js:5187:35)
    at injectionError (http://localhost:8000/js/vendor.js:2581:90)
    at noProviderError (http://localhost:8000/js/vendor.js:2619:12)
    at ReflectiveInjector_._throwOrNull (http://localhost:8000/js/vendor.js:4061:19)
    at ReflectiveInjector_._getByKeyDefault (http://localhost:8000/js/vendor.js:4100:25)
    at ReflectiveInjector_._getByKey (http://localhost:8000/js/vendor.js:4032:25)
    at ReflectiveInjector_.get (http://localhost:8000/js/vendor.js:3901:21)
    at resolveNgModuleDep (http://localhost:8000/js/vendor.js:10904:25)
    at _callFactory (http://localhost:8000/js/vendor.js:10974:32)
    at _createProviderInstance$1 (http://localhost:8000/js/vendor.js:10918:26)
    at initNgModule (http://localhost:8000/js/vendor.js:10868:28)
    at new NgModuleRef_ (http://localhost:8000/js/vendor.js:11965:9)
    at createNgModuleRef (http://localhost:8000/js/vendor.js:11949:12)
    at Object.debugCreateNgModuleRef [as createNgModuleRef] (http://localhost:8000/js/vendor.js:14234:12)
consoleError @ zone.js:522

```

**Comment by ddimitrioglo** [comment]
Hi @BruceGujinhe, 

Let me check this issue and come back with updates to you.

**Comment by ddimitrioglo** [comment]
HI @BruceGujinhe again!

I had a chance to check your use-case and faced the problem too, then after some investigation I've found out that some of the angular-cli dependency was removed [15 days ago](https://github.com/angular/angular-cli/issues/9276#issuecomment-358786800)

To fix this you just have to:
* `cd deep-microservices-todomvc/src/deep-todomvc/frontend` 
* run `npm install @angular-devkit/core --save-dev`
* run `npm run start` again 

and there will not be any error in console (see attachment below)

P.S. This will be fixed in next release, keep following ;) 



![image](https://user-images.githubusercontent.com/9005667/35753626-264c1720-0869-11e8-84cc-88d65780b275.png)

**Comment by ddimitrioglo** [comment]
To summarize discussion above - here are the steps I've made to make this `todos` woking:

* Installed node 6.10.3 from scratch
* Installed `deepify` globally (`npm install deepify -g`)
* Cloned the repo (`git clone git@github.com:MitocGroup/deep-microservices-todomvc.git`)
* `cd deep-microservices-todomvc/src/deep-todomvc/`
* `npm run prepare-backend`

> At this point you will se the message `Do you want to specify parameters values? (Y/N):` answer Y and set the parameter to `true`

* `npm run prepare-frontend`
* `cd ../`
* `deepify server ./` 

> At this point we have deep-framework up and running (DO NOT stop it)

* opened new terminal window
* `cd deep-todomvc/frontend` (being in `/your/system/deep-microservices-todomvc/src`)
* `npm install @angular-devkit/core --save-dev` 

> this is actually the fix

* `npm run start`
* opened [http://localhost:4200/](http://localhost:4200/)

that's it.

**Comment by ddimitrioglo** [comment]
fixed in [v0.1.1](https://github.com/MitocGroup/deep-microservices-todomvc/releases/tag/v0.1.1)
